llvm/test/tools/sycl-post-link/multiple-filtered-outputs.ll

```diff
@@ -41,27 +41,46 @@
 ; CHECK-ALL-EMPTY:
 
 ; PVC does not support sg8 (=1) or sg64 (=2) 
-; CHECK-PVC:      _0.sym
-; CHECK-PVC-NEXT: _3.sym
-; CHECK-PVC-NEXT: _4.sym
-; CHECK-PVC-NEXT: _5.sym
+; CHECK-PVC:      _intel_gpu_pvc_0.prop|{{.*}}_0.sym
+; CHECK-PVC-NEXT: _intel_gpu_pvc_3.prop|{{.*}}_3.sym
+; CHECK-PVC-NEXT: _intel_gpu_pvc_4.prop|{{.*}}_4.sym
+; CHECK-PVC-NEXT: _intel_gpu_pvc_5.prop|{{.*}}_5.sym
 ; CHECK-PVC-EMPTY:
 
+; RUN: FileCheck %s -input-file=%t_intel_gpu_pvc_0.prop -check-prefix=CHECK-PVC-PROP
+; RUN: FileCheck %s -input-file=%t_intel_gpu_pvc_3.prop -check-prefix=CHECK-PVC-PROP
+; RUN: FileCheck %s -input-file=%t_intel_gpu_pvc_4.prop -check-prefix=CHECK-PVC-PROP
+; RUN: FileCheck %s -input-file=%t_intel_gpu_pvc_5.prop -check-prefix=CHECK-PVC-PROP
+; CHECK-PVC-PROP: compile_target=2|oBAAAAAAAAQauRXZs91ZwV3XwZ3Y
```

Comment:
Note for reviewers about encoding this information into device image properties instead of re-using a dedicated field within device image data structure: as part of our transition to `clang-linker-wrapper` will try and unify our device image descriptors with those used in the upstream. There is an equivalent to our properties in there, but there are no dedicated field for device image target. Therefore, here we are using a mechanism that will be easier to transition later.